src/drivers/wdm/queries/WhiteBoard.ql

```diff
@@ -0,0 +1,14 @@
+/**

```

Comment:
I don't think we should check in "scratch" files like this - please remove it.

Reply:
untracked.